### PR TITLE
REL-3924: Prepare for 2021B

### PR DESCRIPTION
**Pull request by swalker2m**
We're leaving GAIA @ Gemini and the new catalog server for December.  To facilitate the upgrade at a later date this PR:

* Accepts either VO table version 1.2 or 1.3
* Removes GAIA @ Gemini from the manual catalog search list only

### CODE REVIEW

File: bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala [hunk 1]
```diff
@@ -32,12 +37,48 @@ object VoTableParser extends VoTableParser {
...
 
-  private def validate(catalogName: CatalogName, xmlText: String): CatalogProblem \/ Unit =
+  private val VoTableHeader: Regex =
+    """<VOTABLE([^>]*)>""".r.unanchored
+
+  private val MinVersion: VersionToken = VersionToken.unsafeFromIntegers(1, 2)
+  private val MaxVersion: VersionToken = VersionToken.unsafeFromIntegers(1, 3)
+
+  private val VersionRegex: Regex =
+    """version="(\d+\.\d+)"""".r.unanchored
```
Comment (swalker2m) [inline bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala:47]: Regex parsing is kind of low rent but I didn't want to dig up an old OSGi-safe version of Atto or something.

**Review by tpolecat — APPROVED** [review]
:+1: assuming tests pass

File: bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala (hunk 1 above)
Comment (tpolecat) [reply bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala:47]: I want an editor plugin that lets me click on a regex and it takes me to regexper, like [this](https://regexper.com/#version%3D%22%28%5Cd%2B%5C.%5Cd%2B%29%22).

Comment (swalker2m) [reply bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala:47]: Wow, that's excellent.

**Pull request merged**